test_libs/pyspec/tests/phase1/block_processing_phase1/test_phase0_block_processing.py

```diff
@@ -0,0 +1,7 @@
+from tests.phase0.block_processing.test_process_attestation import *
```

Comment:
This is how all tests from Phase 0 are imported into Phase 1. I do not like `*` imports (and they are not PEP8 compliant), but these files seem ridiculous when they are imported function-by-function.

Reply:
What's the need of importing phase 0 tests in `tests/phase1`? I think all the phase 0 tests would be tested with `make test` command?

For your question, we can skip it by:
```suggestion
from tests.phase0.block_processing. test_process_attestation import *  # noqa: F401, F403
```

Reply:
The reason for importing the tests from `phase0/*` into `phase1/*` is that it allows for the phase 0 tests to be run against the phase 1 spec. Unless specified otherwise by a pytest decorator, phase 0 tests will be run against phase 1. This prevent us from breaking things in phase 0 by changing phase 1. My new way of testing builds a completely new `phase1/spec.py` that is entirely independent of `phase1/spec.py`